backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/floatingnotification/internal/BpkFloatingNotification.kt

```diff
@@ -53,17 +55,20 @@ internal fun BpkFloatingNotificationImpl(
     modifier: Modifier = Modifier,
 ) {
 
-    Snackbar(
+    Surface(
```

Comment:
MD3 implementation uses a custom layout inside and has weird content alignment issues.
Since we don't use the snackbar for anything else then colors and spacings after some investigation I decided just to use Surface instead.